_posts/2017-12-30-new-years-rust-a-call-for-community-blogposts.md

```diff
@@ -0,0 +1,73 @@
+
+Last year, the Rust team started a new tradition: defining a roadmap of goals for the 
+upcoming year. We leveraged our 
+[RFC process](https://github.com/aturon/rfcs/blob/roadmap-2017/text/0000-roadmap-2017.md)
+to solicit community feedback. While we got a lot of awesome feedback on that RFC, we'd 
+like to try something new **in addition to** the RFC process: a call for community blog posts
+for ideas of what the goals should be.
+
+## #Rust2018
+
+Starting today and running until the end of January we'd like to ask the community to write
+blogposts reflecting on Rust in 2017 and proposing goals and directions for Rust in 2018. 
+These can take many forms:
+
+- A post on your personal or company blog
+- A Medium post
+- A GitHub gist
+- Or any other online writing platform you prefer.
+
+We're looking for posts on many topics: 
+
+- Ideas for community programs
+- Language features
+- Documentation improvements
+- Ecosystem needs
+- Tooling enhancements
+- Or anything else Rust related you hope for in 2018 :D
+
+A great example of what we're looking for is this post by [@mgattozzi](https://github.com/mgattozzi):
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">New post is up for pushing to use <a href="https://twitter.com/rustlang?ref_src=twsrc%5Etfw">@rustlang</a> for WebAssembly:<br><br>Rust and the case for WebAssembly in 2018<a href="https://t.co/qEp91izSok">https://t.co/qEp91izSok</a></p>&mdash; Michael Gattozzi | マイケル・ガトジイ (@mgattozzi) <a href="https://twitter.com/mgattozzi/status/938115299290861569?ref_src=twsrc%5Etfw">December 5, 2017</a></blockquote>
+
+or this post by [Anonyfox](https://github.com/Anonyfox):
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">I just published “Rust in 2017” <a href="https://t.co/RYWUFsEzps">https://t.co/RYWUFsEzps</a></p>&mdash; Maximilian Stroh (@Hisako1337) <a href="https://twitter.com/Hisako1337/status/814435154764722176?ref_src=twsrc%5Etfw">December 29, 2016</a></blockquote>
+
+You can write up these posts and email them to community@rust-lang.org or tweet them with the
+hashtag `#Rust2018`. We'll aggregate any blog posts sent via email or with the hashtag in one
+big blog post here. 
+
+The Core team will be reading all of the submitted posts and using them to inform the initial
+roadmap RFC for 2018. Once the RFC is submitted, we'll open up the normal RFC process, though
+if you want, you are welcome to write a post and link to it on the GitHub discussion.
+
+
+## Timeline
+
+We hope to get a draft roadmap RFC posted in mid January, so blog posts written before then
+would be the most useful. We expect discussion and final comment period of that RFC to last
+through at least the end of January, though, so blog posts until then will also be considered
+ for ideas.
+
+Dates are likely to change, but this is a general overview of the upcoming process:
+
+- **Dec 30**: call for posts!
```

Comment:
Maybe set this to the day this post is published?

Reply:
lol yes yes 👍